### ISSUE TITLE
M (mix) in compose leaves status bar on bottom/just-above-bottom index line

### ISSUE THREAD

**Issue by nabijaczleweli** [neomutt/neomutt#3310]
## Expected Behaviour

show-log-messages but I was in compose. This is beside the point.

## Actual Behaviour

![image](https://user-images.githubusercontent.com/6709544/159022653-a9cad848-add1-4727-9832-fe9ec61c131a.png)

## Steps to Reproduce

1. Press M on the compose screen: 
![image](https://user-images.githubusercontent.com/6709544/159022714-83726b39-cec5-4b57-8bdb-108dab83f643.png)
2. q out of it:
![image](https://user-images.githubusercontent.com/6709544/159022749-c1fcc20e-f42d-4074-923d-ebcfe23ac176.png)
3. postpone:
![image](https://user-images.githubusercontent.com/6709544/159022921-4f2be1c5-3838-426d-a6f0-509c4af25ea9.png)

## How often does this happen?

- Always

## When did it start to happen?

Always (20211029 and `master`).

## NeoMutt Version

```
NeoMutt 20211029-488-e2d454
Copyright (C) 1996-2020 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 5.10.0-11-amd64 (x86_64)
ncurses: ncurses 6.2.20201114 (compiled with 6.2.20201114)
libidn: 1.33 (compiled with 1.33)
GPGME: 1.14.0-unknown
GnuTLS: 3.7.1
libnotmuch: 5.3.0
storage: tokyocabinet, lmdb
compression: zlib, zstd

Configure options: --asan --mandir=/usr/share/man --libexecdir=/usr/libexec --with-mailpath=/var/mail --gpgme --lua --notmuch --gnutls --gss --idn --mixmaster --sasl --tokyocabinet --lmdb --zlib --zstd --testing {CFLAGS=-g -O3 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-sign-compare} {CXXFLAGS=-g -O3 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-sign-compare}

Compilation CFLAGS: -g -O3 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-sign-compare -std=c99 -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -I/usr/include -I/usr/include/lua5.4 -DNCURSES_WIDECHAR -isystem /usr/include/mit-krb5 -fsanitize=address -O0 -fno-omit-frame-pointer -fno-common

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug 
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color 
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop 
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar 
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash 

Compile options:
  -autocrypt +fcntl -flock -fmemopen +futimens +getaddrinfo +gnutls +gpgme +gss 
  +hcache -homespool +idn +inotify -locales_hack +lua +mixmaster +nls +notmuch 
  -openssl +pgp +regex +sasl +smime -sqlite +sun_attachment 

Devel options:
  asan 

MAILPATH="/var/mail"
MIXMASTER="mixmaster"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

## Extra Info

* Operating System and its version: bullseye
* Were you using multiple copies of NeoMutt at once? no
* Were you using 'screen' or 'tmux'? no
* Is your email local (maildir) or remote (IMAP)? IMAP

**Comment by flatcap** [comment]
Please can you give branch [[devel/mixmaster]](https://github.com/neomutt/neomutt/tree/devel/mixmaster) a try.

I've tested it with:
- `unset mixmaster`
- `set mixmaster="junk"`
- `set mixmaster="mixmaster"` (working command)

Thanks

**Comment by nabijaczleweli** [comment]
Works for me

**Comment by flatcap** [comment]
Thanks, merged.

Fixed in 452ed92fa
